src/ResourceManager/Sql/Commands.Sql/help/AzureRM.Sql.md

```diff
@@ -341,9 +341,6 @@ Specifies that a database uses the auditing policy of its host server.
 ### [New-AzureRmSqlSyncGroup](New-AzureRmSqlSyncGroup.md)
 Creates an Azure SQL Database Sync Group.
 
-### [Set-AzureRmSqlSyncGroup](Set-AzureRmSqlSyncGroup.md)
```

Comment:
These were not removed, they were renamed to ```Update-```.  Need to ensure the new cmdlet files are referenced